crashlytics/src/AndroidImpl.cs

```diff
@@ -141,6 +141,24 @@ internal class AndroidImpl : Impl {
         }, "LogException");
     }
 
+    public override void LogExceptionAsFatal(Exception exception) {
+      var loggedException = LoggedException.FromException(exception);
+      StackFrames frames = new StackFrames();
+      foreach (Dictionary<string, string> frame in loggedException.ParsedStackTrace) {
+        frames.Add(new FirebaseCrashlyticsFrame {
+          library = frame["class"],
+          symbol = frame["method"],
+          fileName = frame["file"],
+          lineNumber = frame["line"],
+        });
+      }
+      // TODO(mrober): Do something for empty stacktrace. Get current stacktrace?
```

Comment:
Do we still need this TODO? If so can we clarify the comment more? By reading I'm not quite sure what is supposed to happen.

Reply:
@mrober tried without stack trace and there is no blame for log only on demand fatals: https://screenshot.googleplex.com/tAZaCTidi3NaWxo